src/clj/muuntaja/format/json.clj

```diff
@@ -35,23 +35,23 @@
           options)
         (.flush output-stream)))))
 
-;; muuntaja.json - experimental
+;; jsonista - experimental
```

Comment:
I went with `json.clj` and `jsonista.clj` for now, with the same variable names in both.